carma-messenger-core/carma-messenger/launch/carma-messenger.launch.py

```diff
@@ -54,15 +54,6 @@ def generate_launch_description():
         description = 'Path of folder on host PC containing route CSV file(s) that can be accessed by plugins'
     )
 
-    # Launch ROS2 rosbag logging
```

Comment:
Do we have to remove this ROS 2 rosbag logging GroupAction?

Reply:
This was being declared twice, there is another ros2 rosbag logging action included here https://github.com/usdot-fhwa-stol/carma-messenger/blob/dbc40722a2248c195aa3d063838f1ceec17d7995/carma-messenger-core/carma-messenger/launch/carma-messenger.launch.py#L93